__tests__/app.test.js

```diff
@@ -0,0 +1,54 @@
+const app = require('../app')
+const db = require('../db/connection')
+const request = require('supertest')
+const seed = require ('../db/seeds/seed')
+const {articleData,commentData, topicData, userData} = require('../db/data/test-data/index')
+
+beforeEach(() =>{
+    return seed({topicData, userData, articleData, commentData})
+})
+afterAll(() =>{
+    db.end()
+})
+
+describe('Endpoint for api/topics',() =>{
+    test('Test that it returns an okay 200 message',() =>{
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+    })
+    test('Expect Test to return an array of topic objects that have slug and descirption keys',() =>{
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body}) =>{
+            expect(body.myTopics).toHaveLength(3)
```

Comment:
Great that you're checking the length of the array so you don't get false positives 🙌 if you accidentally get an empty array